### ISSUE TITLE
Test against GNU/Linux and Windows

### ISSUE THREAD

**Issue by pavel-kirienko** [UAVCAN/pyuavcan#64]
https://forum.uavcan.org/t/pyuavcan-design-thread/504/7?u=pavel.kirienko

**Comment by pavel-kirienko** [comment]
Scott said:

> Just **use a docker container**. We have a docker hub organization setup already and Nunavut and libuavcan already build in containers.

But how does one load a kernel module in a containerized environment?

I just tried AppVeyor as was suggested in the linked thread, turns out they don't support vcan either:

```
modprobe: FATAL: Module vcan not found in directory /lib/modules/4.15.0-1027-gcp
```

We need to do something about this. I would really like to avoid any options involving self-hosted build servers.

**Comment by thirtytwobits** [comment]
Yeah, vcan will have to be natively available. You could use buildkite+EC2 or even AWS codebuild + EC2. We might have to build a new AMI but that would be worth the effort given that I could reuse the AMI for libuavcan builds and reduce my reliance on PI workers.

**Comment by pavel-kirienko** [comment]
Nvm, RTFM helped. Turns out they ship `vcan.ko` in extra packages. This commit made it work both on AppVeyor and Travis: https://github.com/UAVCAN/pyuavcan/commit/15e03415de88150723ad055c525723928667ebf1

Windows is up next. Would anybody like to volunteer?

**Comment by pavel-kirienko** [comment]
Fixed in #71
